### DISCUSSION TITLE
Minor: Cloudflare plugin official name

### DISCUSSION THREAD

**Discussion by fdcastel** [troglobit/inadyn#452] [Q&A]
**Why the Cloudflare plugin official name is `default@cloudflare.com`?** -- [Source](https://github.com/troglobit/inadyn/blob/1e1bc9e137f3458db73485d84b107d5eedabf403/plugins/cloudflare.c#L73)

-----

Using `provider cloudflare` or `provider cloudflare.com` (*) in configuration causes the following entry log to appear in each run:

```
Guessing DDNS plugin 'default@cloudflare.com' from 'cloudflare'
```

To remove this use `provider default@cloudflare.com` which is the "correct" name for this provider.

-----
_(*) seen in most examples, including main `README`._

**Comment by troglobit** [comment]
Backwards compatibility.  Usability-wise things could of course be improved.

**Reply by fdcastel** [reply]
Cutting my teeth with the code, now. But **what about some kind of "alias" for providers?**

E.g. `cloudflare` and `cloudflare.com` could be official aliases for `default@cloudflare.com`.

I know... As I said, minor. 😉

**Reply by troglobit** [reply]
Aliases exist already in the sense that if an exact match is not found, when searching for a provider, a substring search is used.  I think the simplest fix would be to change the log level of that warning from NOTICE to INFO, or similar.

**Reply by fdcastel** [reply]
You are right. Of course! 

Sometimes I forgot the K.I.S.S. principle.
